assets/airports/kjfk.json

```diff
@@ -7,227 +8,565 @@
             },
+            "body": ["MERIT"],
             "exitPoints": {
-                "RNGRR": ["SKORR", ["METSS", "A25+"], "RNGRR"],
-                "YNKEE": ["SKORR", ["CESID", "A25+"], "YNKEE"]
+                "MERIT": [],
+                "PUT": ["PUT"]
+            },
+            "draw": [
+                ["MERIT*", "PUT*"]
+            ]
+        },
+        "SKORR4": {
+            "icao": "SKORR4",
+            "name": "Score Four",
+            "rwy": {
+                "KJFK31L": ["_RWY13RSKORR058"],
+                "KJFK31R": ["_RWY13LSKORR063"]
+            },
+            "body": ["SKORR"],
+            "exitPoints": {
+                "SKORR": [],
+                "YNKEE": [["CESID", "A25+"], "YNKEE", "#172"],
+                "RNGRR": [["METSS", "A25+"], "RNGRR", "#223"]
+            },
+            "draw": [
+                ["SKORR", "YNKEE*"],
+                ["SKORR", "RNGRR*"]
+            ]
+        }
+    },
+    "stars": {
+        "CAMRN4": {
+            "icao": "CAMRN4",
+            "name": "CAMRN FOUR",
+            "entryPoints": {
+              "SIE": ["SIE"]
+            },
+            "body": ["BOTON", "HOGGS", "PANZE", "KARRS", "CAMRN"],
+            "rwy": {
+              "KJFK4L": [],
+              "KJFK4R": [],
+              "KJFK13L": [],
+              "KJFK13R": [],
+              "KJFK22L": [],
+              "KJFK22R": [],
+              "KJFK31L": [],
+              "KJFK31R": []
+            },
+            "draw": [
+                ["SIE*", "BOTON", "HOGGS", "PANZE", "KARRS", "CAMRN"]
+            ]
+        },
+        "IGN1": {
+            "icao": "IGN1",
+            "name": "KINGSTON ONE",
+            "entryPoints": {
+              "IGN": ["IGN"]
+            },
+            "body": ["LOLLY", "DOORE", "LENDY", "LGA"],
+            "rwy": {
+              "KJFK4L": [],
+              "KJFK4R": [],
+              "KJFK13L": [],
+              "KJFK13R": [],
+              "KJFK22L": [],
+              "KJFK22R": [],
+              "KJFK31L": [],
+              "KJFK31R": []
             },
-            "draw": [["_315I243SKORR", "SKORR"], ["_315I238SKORR", "SKORR"], ["SKORR", "METSS", "RNGRR*"], ["SKORR", "CESID", "YNKEE*"]]
+            "draw": [
+                ["IGN*", "LOLLY", "DOORE", "LENDY", "LGA"]
+            ]
+        },
+        "LENDY6": {
+            "icao": "LENDY6",
+            "name": "LENDY SIX",
+            "entryPoints": {
+              "LVZ": ["LVZ"]
+            },
+            "body": ["JENNO", "HARTY", "STW", "LENDY", "LGA"],
+            "rwy": {
+              "KJFK4L": [],
+              "KJFK4R": [],
+              "KJFK13L": [],
+              "KJFK13R": [],
+              "KJFK22L": [],
+              "KJFK22R": [],
+              "KJFK31L": [],
+              "KJFK31R": []
+            },
+            "draw": [
+                ["LVZ*", "JENNO", "HARTY", "STW", "LENDY", "LGA"]
+            ]
+        },
+        "PARCH2": {
```

Comment:
Line 508: PARCH2 is now PARCH3 (holding distance at PVD was changed)

Reply:
done, looks like all the same 👍